ui/app/styles/components/status-menu.scss

```diff
@@ -1,31 +1,19 @@
 .status-indicator-button {
   &[data-status='good'] {
-    .status-indicator-icon-dot {
```

Comment:
we no longer have control over individual paths on the SVG, so keep just the dot color for each